Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,43 @@
+{
+	"@context":{
+		"vocab":"http://temperaturesystems.com/Products/#",
+		"Class01":"vocab:Class01",
+		"Sensor029":"Class01:Sensor029",
+		"Computer101":"Class01:Computer101",
+		"Measurement8401":"Sensor029:Measurement8401",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"User10A":"Computer101:User10A"
+	},
+	"Class01":{					   

```

Comment:
You don't need to define it.

Reply:
You mean the xsd prefix, right? That's what i modified!

Reply:
No, I meant delete "Class01":{ and also the closing "}".
Please, restore the xsd prefix definition.